rpcd/playbooks/roles/rpc_maas/templates/neutron_linuxbridge_agent_check.yaml.j2

```diff
@@ -4,8 +4,8 @@ disabled    : false
 period      : "{{ maas_check_period }}"
 timeout     : "{{ maas_check_timeout }}"
 details     :
-    file    : neutron_service_check.py
-    args    : ["--host", "{{ ansible_nodename }}", "{{ internal_vip_address }}"]
+    file    : run_plugin_in_venv.sh
+    args    : ["neutron_service_check.py", "--host", "{{ ansible_nodename }}", "{{ internal_vip_address }}"]
```

Comment:
We might have to pass {{ maas_venv_enabled }} as an argument here to cover the cases where the deployer might not want the maas plugins in a venv.